src/main/java/teammates/logic/api/Logic.java

```diff
@@ -665,12 +665,14 @@ public CourseSummaryBundle getCourseSummaryWithFeedbackSessions(InstructorAttrib
     /**
      * Updates the details of a course.
      *
-     * @see CoursesLogic#updateCourse(CourseAttributes)
+     * @see CoursesLogic#updateCourse(String, String, String)
```

Comment:
What is this `@see` annotation for? I don't think it's necessary since `CoursesLogic.updateCourse(String, String, String)` is called below, so we can go into its declaration if we need to.

Reply:
Just following the old javadoc. I think it's meant for when the javadoc is generated as html?